index.html

```diff
@@ -1,26 +1,126 @@
+    <title>Youtube Video Page</title>
+    <link type="text/css" rel="stylesheet" href="stylesheet.css" />
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <div class="menu">
+          <button class="menu_button"><span><img src="images/menu-icon.png" alt="menu icon" /></span></button>
+        </div>
+        <div class="logo">
+          <a href="#"><img src="images/youtube-icon.png" height="25" width="80"  alt="youtube icon" /></a>
+        </div>
+        <div class="search-div">
+          <form>
+            <input type="text" name="search" class="input1" placeholder="Search" />
+            <button class="input2"><img src="images/search-icon.png" alt="search icon" /></button>
+          </form>
+        </div>
+        <div class="submenu">
+          <ul>
+            <li><a href="#"><img src="images/video-icon.jpg" alt="video-icon" /></a></li>
+            <li><a href="#"><img src="images/google-menu-icon.png" alt="google menu icon" /></a></li>
+            <li><a href="#"><img src="images/notification-icon.jpg" alt="notification icon" /></a></li>
+            <li><a href="#"><img src="images/a-icon.jpg" class="a-icon" alt="profile icon" /></a></li>
+          </ul>
+        </div>
+        
+        
+      </nav>
+      <main class="main_section">
+        <section class="main">
+          <div >
+            <iframe class="vid" src="https://www.youtube.com/embed/Ve8swq7lisU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          </div>
+          <header >
+            <h2 class="tile">Barcelona Vs Napoli 1-1. All Goals & Extended Highlights</h2>
+            <div class="detail">
+              <div class="views">
+                <p>1,234,990 <b>&#183;</b> 17 Mar 2020</p>
+              </div>
+              <nav class="subnav">
+                <ul class="float_right">
+                  <li><button ><img src="images/like-icon.png" alt="like-icon" /> 28k</button></li>
+                  <li><button ><img src="images/dislike-icon.png" alt="dislike-icon" /> 1.5k</button></li>
+                  <li><button ><img src="images/share-icon.png" alt="share-icon" /> Share</button></li>
+                </ul>
+              </nav>
+            </div>
+            <div class="detail margin_top">
+              <div class="views">
+                <div class="float_left">
+                  <img src="images/avatar.jpg" alt="avatar" height="60" width="60" style="border-radius: 50%;" />
+                </div>
+                <div class="views_s">
+                  <h3>Inside Football</h3>
+                  <p>50.6k Subscribers</p>
+                </div>
+              </div>
+              <div class="views">
+                <button class="float_right">SUBSCRIBE</button>
+              </div>
+            </div>
+          </header>
+        </section>
+        <section class="related_vid">
```

Comment:
I don't think this should be a `section`.
Note that a section element needs a heading element (`h2` - `h6`) for this, why not try using an `aside` element instead? :thinking: 

See this document structure from https://www.w3schools.com/html/html5_semantic_elements.asp

![image](https://user-images.githubusercontent.com/36057474/77067096-afbbc580-69e4-11ea-8f13-c1782f55e9c3.png)

This issue is raised as there's a validator warning about using a section element without heading element, learn more here https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2Fadejam%2FEmbedding-a-video%2FAdd-main-content%2Findex.html

Reply:
Thanks for the review